gst-plugin-togglerecord/src/togglerecord.rs

```diff
@@ -1054,7 +1054,7 @@ impl ToggleRecord {
 
         gst_log!(self.cat, obj: pad, "Handling query {:?}", query);
 
-        stream.srcpad.peer_query(query)
+        stream.srcpad.peer_query(query.into())
```

Comment:
This seems asymmetric to me. If we get a `QueryView`, `Pad::query()` and the like should also take a `QueryView`. After all, `Pad::query()` literally just calls this function here (and some minor other things).

Reply:
But at that point, `QueryView` is used everywhere and `QueryRef` seems useless?

Reply:
It's an option, but I find it convenient to pass a concrete query directly, like in function `ToggleRecord::src_query`:
``` rust
                let mut new_query = gst::Query::new_scheduling();
                let res = stream.sinkpad.peer_query(&mut new_query);
```

Reply:
But I agree this is asymmetric...